sw/device/silicon_creator/rom_ext/rom_ext_virtual_addr.ld

```diff
@@ -0,0 +1,31 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Linker script for an OpenTitan ROM_EXT.
+ *
+ * Portions of this file are Ibex-specific.
+ *
+ * The ROM_EXT is actually kept in flash, rather than ROM. While a ROM_EXT can
+ * be loaded into either Slot A (the start of flash), or Slot B (the start of
+ * the upper half of flash), this linker script only targets Slot A.
```

Comment:
This comment needs to be updated since this script is for address translation/virtual address.